consensus-peer/local-deploy.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+set -e
+
+dest=$1
+count=$2
+
+if test -z "$count"
+then
+    echo "usage: local-deploy.sh <destdir> <nreplica>" >&1
+    exit 1
+fi
+
+
+fail=$((($count - 1)/3))
+
+mkdir $dest
+cd $dest
+
+certtool --generate-privkey --outfile key.pem 2>/dev/null
```

Comment:
FWIW, this command doesn't run on OS X (10.11).

![screen shot 2016-08-10 at 22 31 14](https://cloud.githubusercontent.com/assets/14876848/17577331/432228a4-5f4a-11e6-84d6-5deb5c6509c2.png)

If I'm reading the tool's usage text correctly, the equivalent command is `certtool c o=key.pem`, but it works interactively (unless you set up a [template](http://www.manpagez.com/man/1/certtool/certtool-3.2.4.php)).

So, for those on a Mac, testing this locally seems like a no go. Not a deal breaker, but if you have any suggestions (cf. Vagrant), I'd love to hear them.